src/domain/Game.java

```diff
@@ -154,5 +157,15 @@ public String toString() {
 		builder.append("======================");
 
 		return builder.toString();
+		}
+		private boolean isGameOver() {

```

Comment:
In deze method moet je een `if`-statement schrijven dat `true` teruggeeft als `activePlayers.size()` groter is dan 0 en anders `false` teruggeeft.